### PR TITLE
Added Liquid filter to check for and updated duplicate ID values in the WYSIWYG editor

### PR DESCRIPTION
**Pull request by timwright12**
## Description

We have a reported section 508 scan issue of a duplicate ID value the in WYSIWYG content output on https://va.gov/health-care/health-needs-conditions/military-sexual-trauma/ - this PR adds a liquid filter to check for duplicate IDs in the WYSIWYG content and makes them unique.

## Testing Done

Successful pipeline build and spot checking content pages.

### CODE REVIEW

**Comment by timwright12** [comment]
Seeing if we can get this fixed in the CMS before adding the filter

**Review by Cgfor3 — APPROVED** [review]
👍

**Review by zacharymorel — APPROVED** [review]

File: src/site/filters/liquid.js [hunk 1]
```diff
@@ -753,6 +755,40 @@ module.exports = function registerFilters() {
...
+          });
+          // Return the cleaned HTML.
+          replaced = $.html();
+        }
+      });
+    } // if()
+
+    return replaced;
+  };
+
```
Comment (mpelzsherman) [inline src/site/filters/liquid.js:791]: Please add a test for any new filter functions (see `src/site/filters/liquid.unit.spec.js`).

**Comment by mpelzsherman** [comment]
Based on the Slack convo, it sounds like the team correctly concluded that this PR isn’t needed. We appreciate the initiative here, but in general, I think issues like this should probably go through our normal grooming process.

**Review by mpelzsherman — CHANGES_REQUESTED** [review]
In general, we should not be doing this sort of complex logic to fix issues with CMS data.

I think we should close this PR.

File: src/site/filters/liquid.js (hunk 1 above)
Comment (timwright12) [reply src/site/filters/liquid.js:791]: We're only merging here if this become a recurring problem in the WYSIWYG, but CMS has opened a ticket for it so I'm pulling this back into draft for the time being https://github.com/department-of-veterans-affairs/va.gov-cms/issues/5421

**Pull request closed without merging**